lib/active_admin_import/dsl.rb

```diff
@@ -62,7 +62,8 @@ def active_admin_import(options = {}, &block)
         render template: options[:template]
       end
 
-      action_item :import, only: :index do
+      if_option = options[:if] || true
```

Comment:
please move this to the https://github.com/activeadmin-plugins/active_admin_import/blob/master/lib/active_admin_import/options.rb#L31